spec/models/company_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe Company, type: :model do
+
+  describe 'Factory' do
+    it 'has a valid factory' do
+      expect(create(:company)).to be_valid
+    end
+  end
+
+  describe 'DB Table' do
+    it { is_expected.to have_db_column :id }
+    it { is_expected.to have_db_column :name }
+    it { is_expected.to have_db_column :company_number }
+    it { is_expected.to have_db_column :phone_number }
+    it { is_expected.to have_db_column :email }
+    it { is_expected.to have_db_column :street }
+    it { is_expected.to have_db_column :post_code }
+    it { is_expected.to have_db_column :city }
+    it { is_expected.to have_db_column :region }
+    it { is_expected.to have_db_column :website }
+  end
+
+  describe 'Validations' do
+    it { is_expected.to validate_presence_of :company_number }
+
+    it { is_expected.to validate_length_of(:company_number).is_equal_to(10) }
```

Comment:
Recommend adding some tests of org number validation, since that seems to be very important (tests which confirm failure with bad number, and the reverse).

And, yes, the cucumber scenario tests bad org numbers, so this is not critical right now but would still be useful to add here).